dimitriosv/week4/app/components/List.jsx

```diff
@@ -0,0 +1,52 @@
+import Messages from './Messages.jsx';
+var React = require('react');
+var ReactDOM = require('react-dom');
+var MessageActions = require('../actions/message-actions.js');
+
+var List = React.createClass({
+        getInitialState: function() {
+            return {
+                    inputMessageName:"",
+                    listId:this.props.listId
+                };
+        },
+        render: function() {
+            return  <div className="llist">
+                        <div className="listname" >{this.props.listName}</div>
+                        <div>
+                            <input onChange={this.handleMessageChange} type="text"/>
+                            <button type="button" onClick={this.addMessageChild}>Submit</button>
+                        </div>
+                         {/*console.log(thats how you comment inside return)*/}
+                         <Messages  deleteMessageParent={this.props.deleteMessageParent}
```

Comment:
seems to be a lot of leftovers from prev. assignment `deleteMessageParent` `toggleArchiveMessageParent`.
Both here and in the next files.